src/mon/MgrMonitor.cc

```diff
@@ -895,6 +900,16 @@ bool MgrMonitor::prepare_command(MonOpRequestRef op)
       r = -EINVAL;
       goto out;
     }
+    if (!pending_map.all_support_module(module)) {
+      ss << "module " << module << " does not exist";
+      r = -ENOENT;
```

Comment:
I would agree that disabling a nonexistent module should be an error (in order to detect mistyped module names), but I don't think `all_support_module` is the right check.  We should accept a disable command if *any* mgr has reported the module.

Reply:
@jcsp , I re-checked the all_support_module(), it seems it included all mgrs' reported modules(both active and standby). Could you point out why it is not the right check?

Reply:
@jcsp got it. I updated the fix, please take a look.